preload/resources/modulepreload-iframe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<body>
+    <script>
+        const m = new URL('module1.js', location.href).toString();
+        const observer = new PerformanceObserver(l => {
+            const entries = l.getEntriesByName(m);
+            if (entries.length === 1) {
```

Comment:
What if entries.length is more than 1, is that possible? And just for my knowledge here, it will always at least be one, is that right?

Reply:
It can theoretically be 2 if the module was preloaded, and importing it would not detect the preload and would re-fetch the module.

Reply:
But at this point we haven't even `import`ed it. So how could it ever be 2 at this point?